### DISCUSSION TITLE
How to use the local backend?

### DISCUSSION THREAD

**Discussion by bo0tzz** [k8up-io/k8up#663] [General]
The documentation on the local backend seems sparse at best. My impression is that I could use it to backup to a Kubernetes volume instead of to object storage, but I can't actually find out whether that is what it would do. The [object specification](https://k8up.io/k8up/2.1/references/object-specifications.html#_local) only mentions a `mountPath`: MountPath inside the restic container, but I cannot find anything about creating that mount on the container.

**Comment by SchoolGuy** [comment]
Were you able to solve your problems? If yes a short description would be nice so I could try them out maybe?

**Reply by bo0tzz** [reply]
Unfortunately I haven't figured out how to use this yet.

**Comment by Kidswiss** [comment]
Hi Everyone

The local backend consists currently only of configuration stubs. There's no PVC handling implemented in the actual operator. So it's basically a useless setting right now.

**Comment by ghost** [comment]
Any update on local backend docs?

**Comment by jamesbramich** [comment]
Still broken I believe:

INFO    k8up.restic.restic.RepoInit.restic.stderr       Fatal: create repository at /k8s-pv/k8up-repo failed: mkdir /k8s-pv: permission denied

Not sure  what permissions are needed, or if it just isn't implemented yet. It would be a very handy feature to back up to NAS.

**Comment by galindro** [comment]
It isn't working yet... My goal is to backup a k3s cluster running on an edge device to an icloud drive mounted on the host's filesystem. This lack of proper documentation made me lose quite some time trying to make it work. Kindly ask the developers to at least make a note on the docs that this feature isn't working or remove it from the docs.
